### PR TITLE
boilerplate code for connector

### PR DESCRIPTION
**Pull request by darshan-lukhi**
- Read env variable and validate against elasticsearch
- Generate configuration.json file
- Capability and Health endpoint working

### CODE REVIEW

File: scripts/generate.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+VERSION="${VERSION:-v1.0.0}"
+EXEC_DIR=".codegen/$VERSION"
+
+case $(uname -m) in
+    x86_64)   ARCH="amd64" ;;
+    arm)      ARCH="arm64" ;;
```
Comment (gneeri) [inline scripts/generate.sh:8]: Minor thing: I just tried this on my M2 mac, and it looks like it either needs to be `arm64` or `arm*` to work with these machines.

**Review by gneeri — APPROVED** [review]
This looks great—just the minor point in the generation script.

File: scripts/generate.sh (hunk 1 above)
Comment (darshan-lukhi) [reply scripts/generate.sh:8]: This file is auto generated by the `hasura-ndc-go new` command. Is it good idea to change the auto generated code?
Template file link: https://github.com/hasura/ndc-sdk-go/blob/main/cmd/ndc-go-sdk/templates/new/scripts/generate.sh.tmpl

Comment (gneeri) [reply scripts/generate.sh:8]: I see. Thanks for point that out. I'll take it up to the maintainers of the SDK.

**Pull request merged**